charm/docs/tutorial/getting-started.md

```diff
@@ -43,26 +47,32 @@ storage, Redis or PostgreSQL.
 
 Set the allowed hosts. In this example every host is allowed. For a production environment
 only the used hosts should be allowed.
-```
+
+### Deploy the charms:
+
+```bash
 juju config netbox django-allowed-hosts='*'
 ```
 
 ### Redis
```

Comment:
I would update this header to say 
` ### Deploy Redis`

Reply:
I have updated all the titles of that section